coolpuppy/coolpup.py

```diff
@@ -130,19 +130,20 @@ def load_pileup_df(filename, quaich=False):
         vertical_stripe = []
         horizontal_stripe = []
         coordinates = []
-        try:
-            for i in range(len(data)):
-                vstripe = "vertical_stripe_" + str(i)
-                hstripe = "horizontal_stripe_" + str(i)
-                coords = "coordinates_" + str(i)
-                vertical_stripe.append(f[vstripe][:].toarray())
-                horizontal_stripe.append(f[hstripe][:].toarray())
-                coordinates.append(f[coords][:].astype("U13"))
-            annotation["vertical_stripe"] = vertical_stripe
-            annotation["horizontal_stripe"] = horizontal_stripe
-            annotation["coordinates"] = coordinates
-        except:
-            pass
+        if not skipstripes:
+            try:
+                for i in range(len(data)):
+                    vstripe = "vertical_stripe_" + str(i)
+                    hstripe = "horizontal_stripe_" + str(i)
+                    coords = "coordinates_" + str(i)
+                    vertical_stripe.append(f[vstripe][:].toarray())
+                    horizontal_stripe.append(f[hstripe][:].toarray())
+                    coordinates.append(f[coords][:].astype("U13"))
+                annotation["vertical_stripe"] = vertical_stripe
+                annotation["horizontal_stripe"] = horizontal_stripe
+                annotation["coordinates"] = coordinates
+            except:
```

Comment:
Maybe add a specific exception that you want to ignore?

Reply:
How do you mean?

Reply:
except KeyError, or smth like this... To make sure some other unexpected error is not ignored